Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:carbon
+
+WORKDIR /opt/facility-cache
+
+# Copy source
+COPY . ./
```

Comment:
@Zooloo2014 Is there a difference between . and ./ ? If not could you please change this to . to be consistent with the Validator and Orchestrator.